app/javascript/packs/base.jsx

```diff
@@ -4,6 +4,35 @@ import {
   getInstantClick,
   initializeTouchDevice,
 } from '../topNavigation/utilities';
+import { fetchSearch } from '@utilities/search';
```

Comment:
should this be conditional to `initializeMentionAutocompleteTextArea` being called?

Reply:
good catch - yes!